pagopa/io-app
# feat: [IOCOM-855] Add bottom sheet to display message extra data

This PR depends on https://github.com/pagopa/io-app/pull/5560

## Short description
This PR adds the bottom sheet to display message extra data.

<details open><summary>Details</summary>
<p>

| bs |
| - |
| <img src="https://github.com/pagopa/io-app/assets/29163287/03381ebb-e517-432b-b766-abf818e46203" width="300"/> |

</p>
</details> 

## List of changes proposed in this pull request
- added useMessageMoreDataBottomSheet hook to display the bottom sheet
- updated locales
- updated tests

## How to test
Using `io-dev-api-server`, generate a message. Activate experimental ds and open a message. Tap on _show more data_ action and check that the bottom sheet is displayed correctly.
